cypress/locators/show-edit-pod/index.js

```diff
@@ -1,22 +1,36 @@
 import {
-  SHOW_EDIT_POD_TITLE,
-  SHOW_EDIT_SAVE_BUTTON,
-  SHOW_EDIT_CANCEL_BUTTON,
-  SHOW_EDIT_DELETE_BUTTON,
-  SHOW_EDIT_POD_EDIT,
-  SHOW_EDIT_POD_CONTENT_BOX,
+  SHOW_EDIT_POD,
+  CONTENT_SHOW_EDIT_POD,
+  CONTENT_FORM_SHOW_EDIT_POD,
+  HIDE_DELETE_BUTTON_SHOW_EDIT_POD,
+  EDIT_FIELDS_SHOW_EDIT_POD,
+  FORM_FOOTER_SHOW_EDIT_POD,
+  CANCEL_BUTTON_SHOW_EDIT_POD,
+  SAVE_BUTTON_SHOW_EDIT_POD,
+  DELETE_BUTTON_SHOW_EDIT_POD,
+  TITLE_SHOW_EDIT_POD,
+  EDIT_CONTAINER_SHOW_EDIT_POD,
+  UNDO_BUTTON_SHOW_EDIT_POD,
+  TRANSITION_NAME_SHOW_EDIT_POD,
 } from "./locators";
 
 // component preview locators
-export const showEditPodTitle = () => cy.get(SHOW_EDIT_POD_TITLE);
-export const showEditPodSaveButton = () => cy.get(SHOW_EDIT_SAVE_BUTTON);
-export const showEditPodCancelButton = () => cy.get(SHOW_EDIT_CANCEL_BUTTON);
-export const showEditPodDeleteButton = () => cy.get(SHOW_EDIT_DELETE_BUTTON);
-export const showEditPodEdit = () => cy.get(SHOW_EDIT_POD_EDIT);
-export const showEditPodCollapsibleInnerContent = (element, index) =>
-  cy
-    .get(SHOW_EDIT_POD_CONTENT_BOX)
-    .find("div")
-    .eq(1)
-    .find(`[data-element=${element}]`)
-    .eq(index);
+export const showEditPod = () => cy.get(SHOW_EDIT_POD).find("div");
+export const ContentShowEditPod = () => cy.get(CONTENT_SHOW_EDIT_POD);
```

Comment:
Why have you changed the naming convention from lower camel case (showEditPod) to upper camel case (ContentShowEditPod) and switched the componentElement to ElementComponent?